packages/blockchain-tools/src/contracts/soroban/credits/src/contract.rs

```diff
@@ -26,7 +29,10 @@ impl Credits {
     provider: Address,
     vendor: Address,
     bucket: i128,
-    xlm: Address
+    xlm: Address,
+    carbonSac: Address,
+    sink: Address,
```

Comment:
We're going to have more variables with `sink` in the name. Best rename to `sinkContract` or something similar.